### PR TITLE
perf(storage): Use ToListAsync() instead of ToArrayAsync(), because ToArrayAsync() is a wrapper around ToListAsync()

### PR DESCRIPTION
**Pull request by netcore-jroger**

### CODE REVIEW

File: src/Mocha.Storage/EntityFrameworkCore/Jaeger/EFJaegerSpanReader.cs
Comment (eventhorizon-cli) [inline src/Mocha.Storage/EntityFrameworkCore/Jaeger/EFJaegerSpanReader.cs:1]: ToArrayAsync on line 27 has not been replaced.

Comment (netcore-jroger) [reply src/Mocha.Storage/EntityFrameworkCore/Jaeger/EFJaegerSpanReader.cs:1]: I've fixed it.

Comment (eventhorizon-cli) [reply src/Mocha.Storage/EntityFrameworkCore/Jaeger/EFJaegerSpanReader.cs:1]: create another pr if you want to update sth. else

Comment (netcore-jroger) [reply src/Mocha.Storage/EntityFrameworkCore/Jaeger/EFJaegerSpanReader.cs:1]: ok

**Pull request closed without merging**